dashboard/config/linux/bits/kcsan.yml

```diff
@@ -16,3 +16,4 @@ config:
  - KCSAN_REPORT_VALUE_CHANGE_ONLY
  - KCSAN_ASSUME_PLAIN_WRITES_ATOMIC
  - KCSAN_IGNORE_ATOMICS
+ - KCSAN_PERMISSIVE: [v5.14]
```

Comment:
Mainline only changes its major.minor version in Makefile until actual release by Linus.

So even if 5.15-rc1 is out, this will appear as 5.14. I'm planning to switch it once 5.15 final is released.